roles/openshift_health_checker/openshift_checks/package_version.py

```diff
@@ -9,6 +9,9 @@ class PackageVersion(NotContainerizedMixin, OpenShiftCheck):
     name = "package_version"
     tags = ["preflight"]
 
+    recommended_ovs_version_os_3_5 = "2.6"
+    recommended_ovs_version = "2.4"
```

Comment:
We can only expect the combinations of ovs and ocp to become more complicated over time.